frontend/openchat-client/src/openchat.ts

```diff
@@ -5139,6 +5138,14 @@ export class OpenChat extends OpenChatAgentWorker {
         );
 
         cryptoLookup.set(cryptoRecord);
+
+        window.setTimeout(this.refreshBalancesInSeries, 0);
+    }
+
+    private async refreshBalancesInSeries() {
```

Comment:
This raised a bit of an eyebrow. Any particular reason that we want / need to do them one at a time?

Reply:
Yes, so as not to block the browser connection pool with low priority calls